code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -301,8 +302,8 @@
 	switch(progress_in_seconds)
 		if(-INFINITY to GATEWAY_REEBE_FOUND)
 			if(!second_sound_played)
-				for(var/V in GLOB.generic_event_spawns)
-					addtimer(CALLBACK(src, .proc/open_portal, get_turf(V)), rand(100, 600))
+				for(var/i in 1 to PORTAL_COUNT)
```

Comment:
this is a major rebalance to clockcult. went from having a few hundred portals around the station to only 20.

Reply:
Not an intentional rebalance. Should I determine the number of portals based on the number of floor turfs on the station z level or something?